src/parser/transformation.ts

```diff
@@ -87,9 +68,7 @@ export const transformAsset = (
     tokenType: row.token_type,
   };
 
-  toPayment(prePayment, tokenInfoProvider, ensResolver)
-    .then((row) => callback(null, row))
-    .catch((reason) => callback(reason));
+  return toPayment(prePayment, tokenInfoProvider, ensResolver);
```

Comment:
Previously the parsing worked differently due to how fast-csv was structured. 
It basically had two steps:
1) transformation
2) validation

But for async transformation is used a callback which was passed into the transformation function.

The new flow is more direct so instead of calling the callback we can just return the Promise with the result.